### PR TITLE
Fixed importing errors in tests.

### PR DESCRIPTION
**Pull request by sdhutchins**
- Simply updated the import for the tests so the tests would at least run (even though there are errors).

I've linked this PR to an issue as well.

### CODE REVIEW

**Review by JonnyTran — APPROVED** [review]
These import paths looks good to me.

**Review by JonnyTran — APPROVED** [review]

**Review by JonnyTran — APPROVED** [review]
we might as well remove `generate_LncRNA2Target` since their database is no longer publicly available or actively updated (though I have a copy of their data).

**Review by JonnyTran — APPROVED** [review]
agree with these changes. I've actually made the same updates in master as well

**Review by JonnyTran — CHANGES_REQUESTED** [review]
This had an error at `index="gene_name"` because I recently changed the function to take in a `list` or `pd.Index` of gene ids/names, not just the string of `index name`.
I'd still keep `data.initialize_annotations()` if possible, since we need make sure `data.annotations` df gets instantiated. In the future we might want `MessengerRNA.__init__()` to automatically run `initialize_annotations()` so the users doesn't have to remember run it before they `.annotate_attributes()`.

File: tests/test_multiomics.py [hunk 1]
```diff
@@ -35,8 +35,6 @@ def generate_TCGA_LUAD_MessengerRNA_dask():
         npartitions=4,
     )
     data.drop_genes(data.expressions.columns[50:])
-    data.initialize_annotations(index="gene_name", )
```
Comment (JonnyTran) [inline tests/test_multiomics.py:38]: see https://github.com/JonnyTran/OpenOmics/pull/167#pullrequestreview-1124082309

**Comment by sdhutchins** [comment]
Hey! I'm going to work more on the testing and whatnot. Grad school got me sidetracked a bit! Excited to work more on this.

**Comment by JonnyTran** [comment]
> Hey! I'm going to work more on the testing and whatnot. Grad school got me sidetracked a bit! Excited to work more on this.

No problem, totally understand! This is a side project of mine too, but I'm so glad you're interested on working on it!

As far as the current development state, I make new contributions to OpenOmics whenever I need it to do something to support my research work. But, I'm hoping to learn more about how others (like you) uses it for your own research, so it'll be more useful for the overall bioinf research community. 

When you have time, can you list some new features you'd like to see or pain-points you'd like to be fixed? I've made a project board at https://github.com/JonnyTran/OpenOmics/projects/2. You can drop a TODO list there, and I'll go over it.

If you'd like to chat over a Slack group, I can make one too!

**Comment by sdhutchins** [comment]
Hey @JonnyTran - I had some life stuff happen, but I am back into things now. If you have an PRs or whatnot that you want reviewed, feel free to send them my way as well. I'd love to discuss more on how I'm using this in my work.

**Comment by JonnyTran** [comment]
Thank you @sdhutchins, let me know you need anything, especially with access to certain databases or certain analytic use cases. I haven't been contributing to the package recently but will get back to it. I can send you an email with a few survey question on usability and user experiences, and we can follow up from there.

**Pull request merged**